docs/containers.conf.5.md

```diff
@@ -487,6 +487,16 @@ Default transport method for pulling and pushing images.
 Maximum number of image layers to be copied (pulled/pushed) simultaneously.
 Not setting this field will fall back to containers/image defaults. (6)
 
+**image_volume_mode**="bind"
```

Comment:
I think that belongs into the `[containers]` table.

Reply:
No it was added to the [engine] table elsewhere.

Reply:
Why is part of `[engine]`?  To me it feels more on the `[containers]` side of things as it controls internals of a container but not of the engine.

Reply:
I hate that there is a difference, since I am debating all the time myself. Here is how the man page describes them.

```
## CONTAINERS TABLE
The containers table contains settings to configure and manage the OCI runtime.

# ENGINE TABLE
The `engine` table contains configuration options used to set up container engines such as Podman and Buildah.
```
Since this option is done by the engine and not the OCI runtime, I think it belongs in the engine.

Reply:
But I can be swayed...